packages/wallet/src/lib/api/account.ts

```diff
@@ -0,0 +1,42 @@
+import { AxiosError } from 'axios'
+import { z } from 'zod'
+import $axios, { ErrorResponse, SuccessResponse } from '../axios'
+
+export const fundAccountSchema = z.object({
+  account: z.string().uuid(),
+  amount: z.coerce.number({
+    invalid_type_error: 'Please enter a valid amount'
+  })
```

Comment:
```suggestion
  amount: z.coerce.number({
    invalid_type_error: 'Please enter a valid amount'
  }).positive()
```
By specifing `positive` it will only accept numbers that are greater than zero.